custom/icds_reports/utils/aggregation_helpers/distributed/agg_dashboard_acitivity.py

```diff
@@ -99,27 +139,35 @@ def create_table_query(self):
 
     def get_last_agg_date(self):
         from custom.icds_reports.models.aggregate import DashboardUserActivityReport
-        result = DashboardUserActivityReport.objects.filter(date__lt=self.date.strftime("%Y-%m-%d")).aggregate(Max('date'))
+        result = DashboardUserActivityReport.objects.\
```

Comment:
nit: i prefer these multiline queries wrapped in parens rather than using `\` but i am not sure how universal that is